modules/ort/src/main/java/org/eclipse/sw360/antenna/ort/resolver/OrtResultArtifactResolver.java

```diff
@@ -112,17 +112,11 @@ private static ArtifactCoordinates mapSimpleCoordinates(String name, String vers
     }
 
     private static Optional<ArtifactFilename> mapFilename(Package pkg) {
-        final Stream<RemoteArtifact> artifacts = Stream.of(pkg.getSourceArtifact(), pkg.getBinaryArtifact());
-
-        return artifacts
-                .filter(a -> !a.getUrl().isEmpty())
-                .findFirst()
-                .map(a -> {
-                    String fileName = a.getUrl();
-                    String hash = a.getHash().getValue();
-                    String hashAlgorithm = a.getHash().getAlgorithm().toString();
-                    return new ArtifactFilename(fileName, hash, hashAlgorithm);
-                });
+        // Antenna's artifact filename (or filename entries) are meant to refer to binary artifacts, so only use the
+        // filename from ORT's binary artifact URL here.
+        return Optional.of(new File(pkg.getBinaryArtifact().getUrl()).getName())
```

Comment:
doesn't the binary artifact also have a hash? 
In that case you could also give this to the filename

Reply:
You're right, I dropped too much of the original code! Thanks for pointing that out, fixed.